react/components/ColorPicker/color-picker.global.css

```diff
@@ -0,0 +1,58 @@
+/**
+* Vtex ColorPicker
```

Comment:
```suggestion
* VTEX ColorPicker
```

Como sugere o guia da marca:
![image](https://user-images.githubusercontent.com/284515/51609097-5f3ff980-1f00-11e9-897a-90d5e5b24072.png)

http://brand.vtex.com/identity/#writing

Reply:
pra falar bem a verdade acho que nem precisaria ter "VTEX" escrito no nome do componente né? Todos componentes do Styleguide são "VTEX".

Reply:
Na verdade isso não é o nome do componente, é um comnetario no `.css`. Mas devo tirar o "VTEX"?

Reply:
@JuanAlmeida12 desculpa, olhei rapido e nao vi que arquivo era. Meio que tanto faz, então. Mas continua soando estranho botar VTEX antes do nome do componente, continua.